godot-macros/src/godot_api.rs

```diff
@@ -46,23 +48,25 @@ pub fn transform(input_decl: Declaration) -> Result<TokenStream, Error> {
 
 /// Attribute for user-declared function
 enum BoundAttrType {
-    Func(AttributeValue),
-    Signal(AttributeValue),
-    Const(AttributeValue),
+    Func { rename: Option<String> },
+    Signal,
+    Const,
 }
 
-struct BoundAttr {
-    attr_name: Ident,
+struct BoundAttr<'a> {
+    attr_name: &'a str,
     index: usize,
     ty: BoundAttrType,
 }
```

Comment:
Why introduce a lifetime here? It's not like we can't affort to create an `Ident` in this place.

Reply:
It was changed since I couldn't find a way to easily create an `Ident` since the `KvParser::handle_expr` method produces a `TokenStream` not an `Ident`.

I now have found the `util::ident` fn, so I've changed it back to an ident.